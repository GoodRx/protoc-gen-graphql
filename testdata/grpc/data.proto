syntax = "proto3";

package protoc_gen_graphql.test.grpc;

message RequestB {
  string id = 1;
}

message ResponseB {
  string id = 1;
}
