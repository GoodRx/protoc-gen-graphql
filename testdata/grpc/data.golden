# DO NOT EDIT! Generated by protoc-gen-graphql.

type ProtocGenGraphqlTestGrpc_RequestB {
  id: String!
}

input ProtocGenGraphqlTestGrpc_RequestB_Input {
  id: String!
}

type ProtocGenGraphqlTestGrpc_ResponseB {
  id: String!
}
